js/shopping-list.js

```diff
@@ -0,0 +1,50 @@
+var ulElement = document.querySelector('ul');
+
+function addListItems(event){
+    var form = event.target;
+    var items = form.querySelector('#items').value.split(',');
+
+    if(items[0].length > 0){
+        document.querySelector('#user-warning').textContent = "";
+        for(var i = 0; (i < items.length) && (items[i].length > 0) && (items[i] !== ' '); i++){
+            appendItemToList(createXImgElement(), items[i]);
+        }
+    } else {
+        document.querySelector('#user-warning').textContent = "Please enter an item.";
```

Comment:
We have mixed use of single and double-quotes for Strings in this file - either is fine, but we should be consistent.